study-builder/studies/testboston/i18n/es.conf

```diff
@@ -8,7 +8,7 @@
   },
   "consent": {
     "name": "Formulario de consentimiento",
-    "title": "Proceso de consentimiento<a class=\"button button_download\" href=\"consent.pdf\" target=\"_blank\"><img src=\"assets/images/download.svg\" alt=\"Descargar\" class=\"activity-header__icon\">Descargar ficha informativa</a>",
+    "title": "Proceso de consentimiento<a class=\"button button_download\" href=\"es/consent.pdf\" target=\"_blank\"><img src=\"assets/images/download.svg\" alt=\"Descargar\" class=\"activity-header__icon\">Descargar ficha informativa</a>",
```

Comment:
The same thing which I mentioned in the https://github.com/broadinstitute/ddp-angular/pull/200 - should we internationalize PDFs' names?